vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -2042,7 +2042,7 @@ class RoomDetailFragment @Inject constructor(
             VectorCallActivity.newIntent(
                     context = requireContext(),
                     callId = call.callId,
-                    roomId = call.roomId,
+                    roomId = call.signalingRoomId,
```

Comment:
maybe rename `roomId` to `signalingRoomId`?

Reply:
ok